### ISSUE TITLE
Blank Pages: Improper handling of the errors

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#203]
## Expected Behavior

 If something goes wrong, the user should get the proper and clear notification. Its text should be short and contain all necessary information about the nature of the error and the ways of its possible removal.

## Current Behavior

Unexpected errors are not handled.

without a proper error handler:
- If a unexpected errors occurs in a critical-flow (i.e. creating a course, aswering a test), it could data loss among others problems.
- accessibility fault
- users do not have all necessary information to avoid, correct, or report the error 

## Possible Solution or Implementation

https://www.smashingmagazine.com/2020/06/react-error-handling-reporting-error-boundary-sentry/
